test/helper.js

```diff
@@ -33,7 +33,7 @@ if (!cfg.get('azure:accountKey') ||
 var webServer = null;
 mocha.before(async () => {
   webServer = await server('test');
-  webServer.setTimeout(500);
+  webServer.setTimeout(5000);
```

Comment:
The tests are consistently failing on travis due connection dropping. The tests started failing when this timeout was added. This was not noticed, partly because there was no badge to show that the CI was failing. Increasing this timeout fixes the tests.

See https://travis-ci.org/taskcluster/taskcluster-auth/builds. All of the failures are caused by this timeout.